Components/Card/card.css

```diff
@@ -0,0 +1,92 @@
+@import url("/Util/util.css");
+
+.row {
+    display: flex;
+    flex-wrap: wrap;
+    /* align-items: center;
+    height: 700px; */
```

Comment:
It's always a good practice to remove commented-out code to avoid distraction to the reviewer.

Reply:
yes absolutely, I will remove the commented code and refactor it.